content/setting-up-services/how-to-set-up-services/leverage-service-commands.md

```diff
@@ -25,6 +33,13 @@ stages roughly represent the following purposes:
 - **build** - Run commands that build or generate the actual site. This might include things like compiling Sass files,
   updating 3rd party libraries, or running database updates that the current code in the preview depends on.
 
+At any point during the build process, you can use the `ready` command to indicate that a service is "ready" before
```

Comment:
This is confusing to me. I'm not sure what you mean by "At any point during the build process"

This paragraph is in a different format than all of the other command groups, so it's not clear that it is another command group, or that it is a peer to the other command groups in config.yml.